public/raven.min.js

```diff
@@ -0,0 +1,3 @@
+//# sourceMappingURL=raven.min.js.map
```

Comment:
Another way would be to use another route, i.e. '/scripts', as an alias for including the file straight from the node module.